liferay-lima/liferay-portal
# LPS-105422 Localize Display Page Templates URLs in SEO metadata.

**Motivation**

For SEO, Liferay should correctly communicate the URLs generated and for example, the friendly URLs are localized, so that info should be filled correctly.

**Proposed Solution**

1. Use  AssetDisplayPageFriendlyURLProvider to calculate URLs.
2. Maybe in the future, we can move the FriendlyURLMapper to the API, but for now, I think this approach is OK

**How to test it**

1. Create a Web Content
2. Translate it into Spanish
3. Visit the web content display page as an anonymous user.
4. The generated URLs should be translated (You can obtain more info following the steps in https://issues.liferay.com/browse/LPS-105422).
